app/assets/javascripts/client/inventories/InventoryDataEntryCtrl.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .controller('InventoryDataEntryCtrl', InventoryDataEntryCtrl);
+
+  InventoryDataEntryCtrl.$inject = [
+    '$scope',
+    '$modal'
+  ];
+
+  function InventoryDataEntryCtrl($scope, $modal) {
+    var vm = this;
+
+    vm.showInventoryDataEntryModal = function() {
+      vm.modalInstance = $modal.open({
+        template: '<inventory-data-entry-modal inventory="inventory" resource="resource"></inventory-data-entry-modal>',
```

Comment:
It's pulling it from the parent scope (`inventoryDataEntry`'s: `<inventory-data-entry inventory="inventory" resource="resource"></inventory-data-entry>`). This is the same method I used for product entries, which in turn was adapted from your at https://github.com/MobilityLabs/pdredesign-server/blob/5ae5c17ed45ef0ee97ade3eed2b600186f4e3eeb/app/assets/javascripts/client/permissions/AddNewParticipantsCtrl.js.